src/io.cpp

```diff
@@ -120,4 +120,20 @@ _IRQL_requires_max_(DISPATCH_LEVEL) VOID IofCompleteRequest(_In_ PIRP irp, _In_
 {
     UNREFERENCED_PARAMETER(irp);
     UNREFERENCED_PARAMETER(priority_boost);
+}
+
+void
+IoBuildPartialMdl(
+    _In_ PMDL source_mdl,
+    _Inout_ PMDL target_mdl,
+    _Inout_ PVOID virtual_address,
+    _In_ ULONG length
+    )
```

Comment:
```suggestion
    _In_ ULONG length)
```
Follow style in this file of closing paren at end of previous line